.travis.yml

```diff
@@ -9,9 +9,12 @@ before_install:
     - gem install octokit
     - .github/runchecks
     - docker pull ${OS_TYPE}
-    - docker run -it -d -h testdev.pbspro.com --name testdev -v `pwd`:`pwd` --privileged -w `pwd` ${OS_TYPE} /bin/bash
+    - docker run -it -d -h testdev.pbspro.com --name testdev -v `pwd`:`pwd` --privileged -w `pwd` ${OS_TYPE} /bin/sh
     - docker ps -a
     - export DOCKER_EXEC="docker exec -it testdev"
+    - export DOCKER_TESTDEV_IP="$(docker inspect -f '{{ .NetworkSettings.IPAddress }}' testdev)"
+    - ${DOCKER_EXEC} /bin/bash -c "echo '${DOCKER_TESTDEV_IP}     testdev' >> /etc/hosts"
```

Comment:
@kjakkali Thanks for review. I have replaced bash with sh. Please have a look again.